src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -268,86 +267,42 @@ def generate_config(
             self,
             daemon_spec: CephadmDaemonDeploySpec,
     ) -> Tuple[Dict[str, Any], List[str]]:
+
         assert self.TYPE == daemon_spec.daemon_type
-        deps = []  # type: List[str]
 
-        # scrape mgrs
-        mgr_scrape_list = []
-        mgr_map = self.mgr.get('mgr_map')
-        port = cast(int, self.mgr.get_module_option_ex(
-            'prometheus', 'server_port', self.DEFAULT_MGR_PROMETHEUS_PORT))
-        deps.append(str(port))
-        t = mgr_map.get('services', {}).get('prometheus', None)
+        t = self.mgr.get('mgr_map').get('services', {}).get('prometheus', None)
```

Comment:
Is Prometheus defined in the mgr_map? I thought that was only for Ceph-native services...